packages/shadydom/src/utils.js

```diff
@@ -230,3 +230,35 @@ export const assign = (target, source) => {
 export const arrayFrom = (object) => {
   return [].slice.call(/** @type {IArrayLike} */(object));
 };
+
+/**
+ * Converts a single value to a node for `convertNodesIntoANode`.
+ *
+ * @param {*} arg
+ * @return {!Node}
+ */
+const convertIntoANode = (arg) => {
+  return !(arg instanceof Node) ? document.createTextNode(String(arg)) : arg;
```

Comment:
FYI, I removed this explicit conversion to string because browsers don't seem to do this and will throw when trying to `append` a symbol.